.github/workflows/python.yaml

```diff
@@ -0,0 +1,31 @@
+name: Python
+
+on: [push]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.10.4
+      - name: Install dependencies
+        run: make deps
+      - name: Run tests
+        run: make test
+  pythonlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: pip install --upgrade flake8
```

Comment:
Can these run statements be added to the make file and called from here?
We then could use the make commands in our local dev